### PR TITLE
md/update gas EF for eu

### PR DESCRIPTION
**Pull request by mathilde-daugy**
## Issue

Fixes issue [1878](https://linear.app/electricitymaps/issue/ELE-1878/suspiciously-high-co2-intensity-for-gas-in-ro)

## Description
Gas emission factor for Romania was way too high because of undetected outliers
We have now set the upper limit to 800 gCO2eq/kWh for gas
This also makes the EU average decrease hence the modification of EF for zones which use the default EU average

### Preview

See this [sheet](https://docs.google.com/spreadsheets/d/1dITlaKmWC5E21M-CTRhQzgNqjPz7YjqW7U6dGEQOf9A/edit?usp=sharing) for updated factors

### Double check

- [x] I have tested my parser changes locally with `poetry run test_parser "zone_key"`
- [x] I have run `pnpx prettier --write .` and `poetry run format` to format my changes.

### CODE REVIEW

File: config/zones/FI.yaml [hunk 1]
```diff
@@ -53,7 +53,7 @@ emissionFactors:
     gas:
       datetime: '2021-01-01'
       source: EU-ETS, ENTSO-E 2021
-      value: 684.2607818
```
Comment (VIKTORVAV99) [inline config/zones/FI.yaml:56]: Any reason for why the change here is so massive?

File: config/zones/FO-MI.yaml
Comment (VIKTORVAV99) [inline config/zones/FO-MI.yaml:1]: The FO subzone numbers seems to have increased but we are missing the change to the FO.yaml file as well. (will be identical numbers to the subzones but needed for the frontend to represent the new values in aggregated mode)

**Review by VIKTORVAV99 — CHANGES_REQUESTED** [review]
Only thing blocking this from my side is that we need to update the FO.yaml file as well.

File: config/zones/FO-MI.yaml
Comment (mathilde-daugy) [reply config/zones/FO-MI.yaml:1]: We were previously using the gas EF from DK and I think it makes more sense to use the EU average maybe? especially since they do not have gas generation.
What do you think

File: config/zones/FI.yaml (hunk 1 above)
Comment (mathilde-daugy) [reply config/zones/FI.yaml:56]: there was also probably one power plant with a suspiciously high EF which we have now removed from the set

File: config/zones/FO-MI.yaml
Comment (VIKTORVAV99) [reply config/zones/FO-MI.yaml:1]: Yeah I think it does so in that regard it looks great but the FO.yaml file is still using the old DK numbers. So that needs a bump as well.

Comment (mathilde-daugy) [reply config/zones/FO-MI.yaml:1]: ah i didn't realise this. I'll just update the FO files with the numbers so they are consistent

**Review by VIKTORVAV99 — APPROVED** [review]
LGTM! 🎉

Nice to see we are getting even more accurate emission factors!
